<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><!-- Mirrored from bioinfo.mc.vanderbilt.edu/icibm2013/workshop.html by HTTrack Website Copier/3.x [XR&CO'2013], Mon, 07 Apr 2014 15:01:52 GMT --><head><title>International Conference on Intelligent Biology and Medicine (ICIBM 2015)</title><link rel="shortcut icon" type="image/x-icon" href="" /><meta http-equiv="Content-Language" content="English" /><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><link rel="stylesheet" type="text/css" href="style.css" media="screen" /><script src="js/dropdowntabs.js" type="text/javascript"></script><style type="text/css">.style1 {	text-align: center;}.style2 {	text-align: right;}.style4 {	border: 1px solid #E1E1E1;	background-color: #F0F0F0;}.style5 {	text-align: left;}</style></head><body><div id="wrap"><div id="top"></div><div id="content">	<div class="header">	<h1><font color=white>International Conference on Intelligent Biology and Medicine (ICIBM 2015) </h1>	<h2></h2>	<h3><b>November 13-15 , 2015, Indianapolis, IN, USA</b></font></h3>	<h4></h4>	</div>	<div class="breadcrumbs">			<!--iframe name="I1" id="I1" src="images/menutop.html" style="width: 100%; height: 50px" marginwidth="2" marginheight="1" scrolling="no" border="0" frameborder="0" target="_parent">			Your browser does not support inline frames or is currently configured not to display inline frames.			</iframe-->				<div id="tabsJ">					 <div id="bluemenu" class="bluetabs">							<ul>													<li><a href="index.html"><span>Home</span></a></li>								<li><a href="#" rel="dropmenugs_pp"><span>Submission</span></a></li>								<li><a href="dates.html"><span>Important Dates</span></a></li>								<li><a href="registration.html"><span>Registration</span></a></li>								<li><a href="#" rel="dropmenugs_p"><span>Program</span></a></li>								<li><a href="organization.html"><span>Organization</span></a></li>								<li><a href="invitation.html" rel="dropmenugs_t"><span>Travel</span></a></li>								<li><a href="sponsors.html"><span>Sponsors</span></a></li>								<li><a href="contact.html"><span>Contact</span></a></li>								<!--li><a href="http://bioinfo.mc.vanderbilt.edu/icibm2013/news.html"><span>News</span></a></li-->								<li><a href="#" rel="dropmenugs_c"><span>Past ICIBM</span></a></li>							</ul>			  </div> 		</div>    	    	       <div id="dropmenugs_p" class="dropmenudiv_b">					  <a href="news.html">What's New</a>				    <a href="schedule.pdf" target="_blank">Schedule</a>				    <a href="workshop.html">Workshop</a>           <a href="tutorial.html">Tutorial</a>         </div>	                                    <div id="dropmenugs_pp" class="dropmenudiv_b">					<a href="call.html">Call for Papers</a>				    <a href="submission.html" >Submission</a>	        </div>                 	        <div id="dropmenugs_t" class="dropmenudiv_b">					  <a href="hotel.html">Hotel</a>				    <a href="information.html">About San Antonio</a>	        </div>        	         	        <div id="dropmenugs_c" class="dropmenudiv_b">										<a href="http://compgenomics.utsa.edu/icibm2014/" target="_blank">ICIBM 2014</a>					<a href="http://bioinfo.mc.vanderbilt.edu/icibm2013/" target="_blank">ICIBM 2013</a>					<a href="http://bioinfo.mc.vanderbilt.edu/icibm/index.html" target="_blank">ICIBM 2012</a>	        </div>  	        	        <script type="text/javascript">tabdropdown.init("bluemenu")</script>	    	    </div>  </div> <center>Available soon</center> <!--  <table style="width: auto" cellspacing="10" cellpadding="5">    <tr>      <td width="1.5%"/>    </tr>    <tr>      <td height="auto"/>      <td/>      <td valign="top">        <p>&nbsp;</p>        <b>          <font color="#292998" size="+1.7">NGS Workshop [Chairs: Kun Huang and Dongxiao Zhu]</font>        </b>        <br>          <font color="#292998">            August 11, 2013          </font>          <p>            <br />            <br />            <b>              <font color="#292998" size="+1.7">Agenda</font>            </b>          </p>          <br>            <table>              <tr>                <td width="130em">                  <b>11:00-11:05 am</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Opening remarks</strong>                </td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <th colspan="3">                  <h3 align="left">                    <font color="#292998">Session I</font>                  </h3>                </th>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>11:05-11:35 am</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Degui Zhi: </strong> "Genotype calling from NGS data: incorporating haplotype information in reads"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>11:35 am-12:05 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Xin Jin: </strong>"Profiling of different types of de novo and rare inherited mutations in autism spectrum disorders revealed by whole genome sequencing"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>12:05-12:35 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Haixu Tang: </strong>Spontaneous mutations in bacteria revealed by whole-genome re-sequencing"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>12:35-1:30 pm</b>                </td>                <td></td>                <td>                  <b>Light Lunch </b>                </td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <th colspan="3">                  <h3 align="left">                    <font color="#292998">Session II</font>                  </h3>                </th>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>1:30-2:00 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Yuzhen Ye: </strong>"Computational approaches for metagenomic mining"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>2:00-2:30 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Maureen Sator: </strong>"PePr: a Peak-calling and Prioritization Pipeline for Identifying DNA-binding Sites in Replicated ChIP-seq Experiments"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>2:30-3:00 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Peilin Jia: </strong>"Personalized mutation network analysis of putative cancer genes from next-generation sequencing data"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>3:00-3:30 pm</b>                </td>                <td></td>                <td>                  <b>Break </b>                </td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <th colspan="3">                  <h3 align="left">                    <font color="#292998">Session III</font>                  </h3>                </th>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>3:30-4:00 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Yunlong Liu: </strong>"Detection of allelic specific expression from RNA-seq data"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>4:00-4:30 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Kun Huang: </strong>"Whole transcriptome analysis in pharmacogenomics applications"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>4:30-5:00 pm</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Lee Cooper: </strong>"Integrating Image Phenotypes with Genomics: Applications in Gliomas"                </td>              </tr>              <tr>                <td/>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>            </table>              <br>                <p>                  <br>              </p>                <h3>                  <font color="#292998">Abstract:</font>                </h3>                <p>                  <br>                </p>                <b>                  <font color="#292998">Genotype calling from NGS data: incorporating haplotype information in reads</font>                </b>                <br><br>                  <p>                    Kui Zhang, Degui Zhi<br>                      Department of Biostatistics, University of Alabama at Birmingham                    </p>                  <br>                    Hidden Markov model (HMM) based on Li and Stephens [1] model that takes into account chromosome sharing of multiple individuals results in mainstream haplotype phasing algorithms for genotyping arrays and next-generation sequencing (NGS) data. However, existing methods based on this model do not consider haplotype informative reads, i.e., reads that cover multiple heterozygous sites, which carry useful haplotype information. In this work, we extend our previous work [2], we developed a new HMM to incorporate a two-site joint emission term that captures the haplotype information across two adjacent sites. While our model improves the accuracy of genotype calling and haplotype phasing, haplotype information in reads covering non-adjacent sites and/or more than two adjacent sites is not used due to the severe computational burden. We develop a new probabilistic model for genotype calling and haplotype phasing from NGS data that incorporates haplotype information of multiple adjacent and/or non-adjacent sites covered by a read over an arbitrary distance. We develop a new hybrid MCMC algorithm that combines the Gibbs sampling algorithm of HapSeq and Metropolis-Hastings algorithm and is computationally feasible. We show by simulation and real data from the 1000 Genomes Project that our model offers superior performance for haplotype phasing as well as genotype calling for population NGS data over existing methods.<br><br>                    References:<br>                    1.	Li, N. and M. Stephens, Modeling linkage disequilibrium and identifying recombination hotspots using single-nucleotide polymorphism data. Genetics, 2003. 165(4): p. 2213-33.<br>                    2.	Zhi, D., et al., Genotype calling from next-generation sequencing data using haplotype information of reads. Bioinformatics, 2012. 28(7): p. 938-46.<br>                    <br/><br/>                      <b>                        <font color="#292998">Profiling of different types of de novo and rare inherited mutations in autism spectrum disorders revealed by whole genome sequencing</font>                      </b>                      <br>                        <br>                          <p>                            Xin Jin<br>                      BGI America                    </p>                          <br>                            Autism Spectrum Disorder (ASD) demonstrates high heritability, familial clustering and ~4:1 male to female bias, yet the genetic causes are only partially understood, due to extensive clinical and genetic heterogeneity. Whole genome sequencing (WGS) promises added value to identify novel ASD risk genes as well as new mutations in known loci, but an assessment of its full utility in an ASD group has not been performed. BGI and collaborators have initiated an international endeavor called the Autism Genome 10K-Project, which aims to sequence the genomes of 10,000 individuals from ASD families. In a pilot study, we used WGS to examine 76 families (33 from China, 33 from the US, 10 from middle east) with ASD to detect different types de novo or rare inherited genetic variants, including SNV, small Indel, CNV, SV and retrotransposon insertion. Among ASD probands, we identified number of deleterious de novo mutations and X-linked or autosomal inherited alterations (some had combinations of mutations). These deleterious mutations variants were found in number of novel, known, and candidate ASD risk genes. Taken together, these results suggest that WGS and thorough informatic analyses for de novo and rare-inherited mutations will improve the detection of genetic variants likely associated with ASD or its accompanying clinical symptoms. <br>                                    <br/>                                    <br/>                              <b>                                <font color="#292998">Spontaneous mutations in bacteria revealed by whole-genome re-sequencing</font>                              </b>                              <br>                                <br>                                  <p>                                    Haixu Tang<br>                      School of Informatics and Computing, Indiana University, Bloomington, IN                    </p>                                  <br>                                    The advance of next-generation sequencing techniques has transformed the research paradigms in many fields in life sciences, including microbiology. The low cost of NGS enables large-scale investigation of genomic variations in bacteria under distinct or no selection pressure. I will report here the findings on the spontaneous mutations in the bacterium E. coli, revealed by whole-genome re-sequencing of mutation-accumulation (MA) E. coli strains, in which selection pressure is minimized. For the first time from genome-wide data, we derived the rate and spectrum of spontaneous mutations in both wide-type and mismatch repair (MMR) deficiency strains in E. coli. We observed the distribution of spontaneous mutations fell into a wave-like spatial pattern that is repeated in the two separately replicated halves of the E. coli chromosome. Finally, I will also report the large-scale genome rearrangement occurring in neutrally evolved E. coli strains.<br><br>                                    This work is collaborated with the labs of Pat Foster and Michael Lynch at Indiana University.                                    <br>                                      <br/>                                      <br/>                                      <b>                                        <font color="#292998">Computational approaches for metagenomic mining</font>                                      </b>                                      <br>                                        <br>                                          <p>                                            Yuzhen Ye<br>                     School of Informatics and Computing, Indiana University, Bloomington, IN                    </p>                                          <br>                            Fueled by advances in next generation sequencing, metagenomics is revolutionizing the field of microbiology, and has excited researchers in many disciplines that could benefit from the study of environmental microbes, by enabling analyses of microbial communities.  Annotating large metagenomic datasets, however, can be challenging, and directed computational approaches are often needed to make the best use of metagenomic datasets. In this talk, I will first introduce FragGeneScan and RAPSearch, tools that we have developed for predicting genes (often fragmented) from metagenomic sequences and fast similarity searches at protein level. I will also talk about targeted computational approaches that we have developed for studying CRISPR–Cas immune systems, demonstrating the importance of targeted analyses of general metagenomic datasets. The CRISPR–Cas adaptive immune system is a primary defense system in prokaryotic organisms, providing targeted defense against invading DNAs (including viruses and plasmids): bacteria memorize invaders by incorporating pieces of the invaders’ sequences, called spacers, into CRISPR (clustered regularly interspaced short palindromic repeats) loci between repeats, forming arrays of repeat-spacer units. Our targeted computational approaches include a targeted assembly approach that greatly improves the identification of CRISPR arrays from metagenomic sequences, and an approach for fishing out the invaders of bacterial communities using spacers extracted from assembled CRISPR arrays. Application of our tools to human microbiomes has revealed a variety of CRISPR–Cas systems and a diverse collection of invasive mobile genetic elements in human microbiomes, which hopefully will be an important resource for studying the interactions between bacteria and invaders.                            <br>                              <br/>                              <br/>                              <b>                                <font color="#292998">PePr: a Peak-calling and Prioritization Pipeline for Identifying DNA-binding Sites in Replicated ChIP-seq Experiments</font>                              </b>                              <br>                                <br>                                  <p>                                    Maureen Sator<br>     Department of Computational Medicine and Bioinformatics, University of Michigan, Ann Arbor, MI    </p>                                  <br>                                    ChIP-seq is now the standard method to identify genome-wide DNA-binding sites for transcription factors (TFs) and histone modifications. As use of this technique grows, there is a growing demand to analyze experiments with biological replicates, especially for epigenomic experiments where variation among biological samples is most significant. However, even with TF binding, the observed variation is higher among replicates than most current peak finders assume.                                    I will present a novel Peak-calling and Prioritization pipeline (PePr) for replicated ChIP-seq experiments. PePr uses a local negative binomial distribution, ranking consistent binding sites more favorably than sites with greater variability. Comparing PePr to commonly-used approaches on several transcription factor ChIP-seq datasets, we show PePr uniquely identifies regions with enriched tag counts, high motif occurrence rate and known characteristics of TF binding based on visual inspection. For histone modification data displaying substantial variation among samples, PePr achieved better specificity than alternative approaches by identifying regions that are more consistently different between the two sample groups. I will illustrate how PePr estimates the shift size to align alternate strand reads, the optimal window size across the genome, the significance and FDR levels, and post-processing steps to remove artifacts and improve peak resolution. PePr is made available as a Google Code project at <a href="https://code.google.com/p/pepr-chip-seq">https://code.google.com/p/pepr-chip-seq</a>.                                    <br>                                      <br/>                                      <br/>                                      <b>                                        <font color="#292998">Personalized mutation network analysis of putative cancer genes from next-generation sequencing data</font>                                      </b>                                      <br>                                        <br>                                          <p>                                            Peilin Jia<br>     Department of Biomedical Informatics, Vanderbilt University School of Medicine, Nashville, TN USA    </p>                                          <br>                                            A major challenge in interpreting the large volume of mutation data identified by next-generation sequencing (NGS) is to distinguish driver mutations from neutral passenger mutations to facilitate the identification of targetable genes and new drugs. Current approaches are primarily based on frequencies, which lack the power to detect infrequently mutated driver genes and ignore functional interconnection and regulation among cancer genes. We propose a novel, personalized mutation network method, VarWalker, which well adjusts personal mutation profile and builds on joint frequency of interacting mutation genes in protein-protein interaction networks. VarWalker fits a sample-specific generalized additive model to estimate the probabilities of mutation events in each patient’s genome. With a weighted resampling procedure, passenger mutations which are largely raised by random events can be removed. VarWalker then examines mutation genes as well as their close interactors by applying the algorithm Random Walk with Restart. The method is further tailored with comprehensive resample- and randomization-based tests for evaluation. In our applications to two large-scale NGS benchmark datasets (183 lung adenocarcinoma samples and 121 melanoma samples), we constructed a functionally connected, cancer-gene driven subnetwork for each disease. Importantly, VarWalker enables identification of not only highly recurrently mutated genes, but also well-studied, yet infrequently mutated genes. Utilizing VarWalker, we demonstrated that network-assisted approaches can be effectively adapted to facilitate detecting driver genes in cancer from NGS data. VarWalker prioritizes cancer genes, either frequently or infrequently mutated, and illustrates their interactions in PPI networks.                                            <br>                                              <p>                                                <br/>                                                <br/>                                                <b>                                                  <font color="#292998">Identifying allele specific expression from RNA-seq data</font>                                                </b>                                                <br>                                                  <br>                                                    <p>                                                      Yunlong Liu<br>     Department of Medical and Molecular Genetics, Indiana University School of Medicine, Indianapolis, IN    </p>                                                    <br>                                                      In this talk, I will discuss our strategy of studying transcriptional regulation using allele specific expression levels derived from RNA-seq experiments. We will demonstrate this strategy using tissue specific transcriptome data derived from F1 crosses between C57BL/6 and DBA/2J.                                                      <br>                                                        <p>                                                          <br/>      <br/>      <b>        <font color="#292998">Whole transcriptome analysis in pharmacogenomics applications</font>      </b>      <br>        <br>          <p>            Kun Huang<br>     Department of Biomedical Informatics, The Ohio State University, Columbus, OH    </p>      <br>        In this talk I will discuss our ongoing work on generating and processing large whole transcriptome data from RNA-seq technology for identifying regulatory elements in pharmacogenomics applications. Using whole transcriptome sequencing with high depth (100-180 million reads per sample), we are able to discover functional SNPs by identifying allelic specific expression sites. In addition, we can identify lncRNAs which show strong correlative relationships with mRNAs and compare them among multiple brain regions between smokers and non-smokers.        <br>         <p>           <p>             <br/>             <br/>             <b>               <font color="#292998">Integrating Image Phenotypes with Genomics: Applications in Gliomas </font>             </b>             <br>               <br>                 <p>                   Lee Cooper<br>     Department of Biomedical Informatics, Emory University, Atlanta, GA    </p>                 <br>                   Computational analysis of pathology and radiology imaging is playing an increasing role in studies of cancer. Combining quantitative measurements of the macro and microscopic properties of tumors with rich genomic and clinical descriptions can provide insights into mechanisms of tumor progression. This talk will present several in silico studies of glioma brain tumors where we have developed computational pipelines to integrate imaging and genomics using data from The National Cancer Institute's Cancer Genome Atlas (TCGA). The TCGA contains a wide array of genomic characterizations of hundreds of patients from more than 20 tumor types, and presents a unique opportunity to link imaging observations to genetics and patient outcome. We show how these approaches can leverage multimodal datasets to link imaging phenotypes to molecular drivers, define image-based subtypes of tumors, and to understand the role of tissue microenvironment in establishing patterns of gene expression.                   <br>                     <p>                       <p>                         <p>                      <br>                  </p>                         <td/>                  </table>-->              <p>&nbsp;</p><p>&nbsp;</p><div id="clear"></div><div id="bottom"></div></div><div id="footer">Hosted by: <a href="http://www.compbio.iupui.edu/" target="_blank">Center For Computational Biology and Bioinformatics, IU School of Medicine</a></div></body><!-- Mirrored from bioinfo.mc.vanderbilt.edu/icibm2013/workshop.html by HTTrack Website Copier/3.x [XR&CO'2013], Mon, 07 Apr 2014 15:01:52 GMT --></html>