<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd"><html><!-- Mirrored from bioinfo.mc.vanderbilt.edu/icibm2013/tutorial.html by HTTrack Website Copier/3.x [XR&CO'2013], Mon, 07 Apr 2014 15:01:52 GMT --><head><title>International Conference on Intelligent Biology and Medicine (ICIBM 2015)</title><link rel="shortcut icon" type="image/x-icon" href="" /><meta http-equiv="Content-Language" content="English" /><meta http-equiv="Content-Type" content="text/html; charset=UTF-8" /><link rel="stylesheet" type="text/css" href="style.css" media="screen" /><script src="js/dropdowntabs.js" type="text/javascript"></script><script src="http://code.jquery.com/jquery-latest.min.js"></script><style type="text/css">.style1 {	text-align: center;}.style2 {	text-align: right;}.style4 {	border: 1px solid #E1E1E1;	background-color: #F0F0F0;}.style5 {	text-align: left;}</style></head><body><div id="wrap"><div id="top"></div><div id="header"></div><script>    $("#header").load("header.html");</script>   <center>Available soon</center>   <!--<table style="width: auto" cellspacing="10" cellpadding="5">  <tr>    <td width="1.5%"/>  </tr>  <tr>    <td height="auto"/>    <td/>    <td valign="top"><p>&nbsp;</p>      <b>        <font color="#292998" size="+1.7">Tutorial I: Introduction to Proteome Informatics [Chair: David Tabb] </font>      </b><br>        <font color="#292998">          August 11, 2013        </font>      <p>  <br />        <br />        <b>          <font color="#292998" size="+1.7">Agenda</font>        </b>      </p>      <br>        <table>          <tr>            <td>              <b>9:00-10:10 AM</b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <strong>Identifying proteins through LC-MS/MS</strong>            </td>          </tr>          <tr>            <td align="left">David Tabb<td/>            <td/>            <td></td>          </tr>          <tr>            <td>              <b></b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <lu><li>Generating tandem mass spectral collections through analytical chemistry</li><li>Identifying peptide matches through database search algorithms</li><li>Thresholding peptide-spectrum matches to a specified False Discovery Rate</li>                <li>Assembling parsimonious protein lists via bipartite graphs</li>              </lu>            </td>          </tr>                              <tr>            <td>&nbsp;</td>          </tr>          <tr>            <td>              <b>10:10-11:20 AM</b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <strong>Comparative and quantitative proteomics</strong>            </td>          </tr>          <tr>            <td align="left">              Yao-Yi Chen<td/>              <td/>              <td></td>            </tr>          <tr>            <td>              <b></b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <lu>                <li>Differentiating cohorts by protein spectral counts</li>                <li>Incorporating MS intensity in protein quantitation</li>                <li>Monitoring batch effects and proteomics quality control</li>              </lu>            </td>          </tr>          <tr>            <td>&nbsp;</td>          </tr>          <tr>            <td>              <b>11:20-11:30 AM</b>            </td>            <td></td>            <td>              <b>Break</b>            </td>          </tr>          <tr>            <td>&nbsp;</td>          </tr>          <tr>            <td>              <b>11:30 AM-12:00 PM</b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <strong>Proteomic repositories and spectral libraries</strong>            </td>          </tr>          <tr>            <td>              Matthew Chambers            </td>            <td/>            <td align="left"></td>          </tr>          <tr>            <td>              <b></b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <lu>                <li>Publishing proteomes through ProteomeXchange and LabCAS</li>                <li>Standardizing data representations</li>                <li>Building upon prior identifications through spectral library search</li>              </lu>            </td>          </tr>          <tr>            <td>&nbsp;</td>          </tr>          <tr>            <td>              <b>12:00-12:30 PM</b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <strong>Proteogenomics through RNA-Seq / proteomic combination</strong>            </td>          </tr>          <tr>            <td>Xiaojing Wang</td>            <td/>            <td align="left"></td>          </tr>          <tr>            <td>              <b></b>            </td>            <td>&nbsp;&nbsp;</td>            <td align="left">              <lu>                <li>Customizing protein sequence databases from RNA-Seq data</li>                <li>Evaluating impact on proteomic identification</li>              </lu>            </td>          </tr>          <tr>            <td>&nbsp;</td>          </tr>         </table><br>        <h3><font color="#292998">Abstract:</font></h3>  Identifying proteins and post-translational modifications (PTMs) from tandem mass spectrometry data depends heavily on an ecosystem of algorithms that has emerged during the last decade.  This workshop introduces major elements of the protein identification and quantitation pipelines and describes a strategy for proteogenomic experiments with both RNA-Seq and proteomic data.  The content is designed to be accessible to computer scientists and bioinformaticists who have not previously worked with proteomics data sets.<br/><br/>          <h3>            <font color="#292998">Instructors:</font>          </h3>            <b>David L. Tabb, Ph.D.,</b>            is an associate professor of Biomedical Informatics at Vanderbilt University.  He has developed algorithms for protein identification since 1996, when he began graduate school in the Laboratory of John Yates.  He contributes bioinformatics expertise for two National Cancer Institute programs in proteomics: the Clinical Proteomics Technology Assessment for Cancer and the Early Detection Research Network.<br>              <br>                    <b>      Yao-Yi Chen, M.S.,    </b> is a Ph.D. candidate with the Department of Biomedical Informatics at Vanderbilt University.  She has applied her Master’s degree in Biostatistics in the area of comparative and quantitative proteomics, with multiple publications and an oral presentation at the prestigious American Society of Mass Spectrometry. <br><br>          <b>Matthew C. Chambers, B.S., </b>is a bioinformatics systems engineer with the Department of Biomedical Informatics at Vanderbilt University.  He plays a key role in the development of the ProteoWizard library that enables support for raw data import from the native file formats of many mass spectrometry vendor.  He has also played a key role in the standardization of file formats for proteomics.<br><br>      <b>Xiaojing Wang, Ph.D., </b>        is a postdoctoral research associate in Dr. Bing Zhang’s group at Vanderbilt University. She joined Dr. Zhang’s group in 2010 with a Ph.D. degree in bioinformatics. She develops bioinformatics tools that utilize proteomic data to facilitate the functional interpretation of genomic variations.<br><br>      <td width="1.5%"></td>  </tr></table>  <table style="width: auto" cellspacing="10" cellpadding="5">    <tr>      <td width="1.5%"/>    </tr>    <tr>      <td height="auto"/>      <td/>      <td valign="top">        <b>          <font color="#292998" size="+1.7">Tutorial II: Pathway and network analysis tutorial [Chair: Alexander Pico] </font>        </b>        <br>          <font color="#292998">            August 11, 2013          </font>          <p>            <br />            <br />            <b>              <font color="#292998" size="+1.7">Agenda</font>            </b>          </p>          <br>            <table>              <tr>                <td>                  <b>1:30-2:20 PM</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Alexander Pico:</strong> "Network visualization and analysis with Cytoscape"                </td>              </tr>              <tr>                <td align="left"><td/>                  <td/>                  <td></td>                </tr>                            <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>2:20-2:30 PM</b>                </td>                <td></td>                <td>                  <b>Break</b>                </td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>2:30-3:20 PM</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Alexander Pico: </strong> "Building pathway models for analysis with WikiPathways"                </td>              </tr>              <tr>                <td align="left"><td/>                  <td/>                  <td></td>                </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>3:20-3:30 PM</b>                </td>                <td></td>                <td>                  <b>Break</b>                </td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>3:30-4:20 PM</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Jing Wang: </strong>"Integrating multidimensional omics data over biological networks with NetGestalt"                </td>              </tr>              <tr>                <td></td>                <td/>                <td align="left"></td>              </tr>              <tr>                <td>&nbsp;</td>              </tr>              <tr>                <td>                  <b>4:20-5:00 PM</b>                </td>                <td>&nbsp;&nbsp;</td>                <td align="left">                  <strong>Alexander Pico/Jing: </strong>Wang Q/A and hands on                </td>              </tr>             </table>            <br>              <p>                <br>              </p>              <h3>                <font color="#292998">Abstract:</font>                              </h3>              <p><br></p>              <b>                <font color="#292998">Network Visualization and Analysis with Cytoscape</font>              </b>              <br>                <p>                  Alex Pico<sup>1</sup><br>                    <sup>1</sup>Gladstone Institutes, San Francisco, CA 94158, USA                  </p>                <br>                  The network perspective on biology aims to bring meaningful context to high-throughput data for exploratory analysis, interpretation and hypothesis generation. As a free and open source tool, Cytoscape has become the most popular network visualization and analysis tools in the biological sciences. It is now cited in over 400 publications per year and downloaded over 7,000 times per month.  This tutorial will provide a general introduction to network biology studies and Cytoscape concepts, including a hands-on session for universal data import and demonstration of a few of the over 100 freely available apps contributed by the Cytoscape developer community. By the end of this tutorial, you should be able to import any tabular data and integrate it with public sources of networks, pathways and other datasets. The tutorial will focus on the 2.8.3 version of Cytoscape (which has the largest pool of apps) while briefly introducing the benefits and major differences of the latest 3.0.1 release. <b>Recommended:</b> Download and install Cytoscape 2.8.3 prior to the tutorial.<br/><br/>                  <p>                    <br>                  </p>                  <b>                    <font color="#292998">Building Pathway Models for Analysis with WikiPathways</font>                  </b>                  <br>                    <p>                      Alex Pico<sup>1</sup><br>                        <sup>1</sup>Gladstone Institutes, San Francisco, CA 94158, USA                      </p>                    <br>                      WikiPathways is a collaborative platform for building, curating and distributing biological pathway knowledge for the research community. This non-traditional approach allows YOU to easily build your own pathway models highlighting the biological processes that are most relevant to your research. The resulting models are then immediately available to you and the community as image files AND as data files that can be loaded into various data visualization and analysis tools, including PathVisio and Cytoscape. The contributed models are available via direct download, web services and a sparql endpoint.  Receiving 25,000 visits per month, WikiPathways is experiencing a steep increase in participation and access. This tutorial will provide a general introduction to the navigation and access tools at WikiPathways, as well as how to get started building and editing pathways of your own. By the end of this tutorial, you should be able to find pathways of interest from multiple starting points and to begin your own collaborative curation project. <b>Recommended:</b> Consider pathways of interest to your research and see if you can find existing images or diagrams online to serve as starting material for a new pathway model.<br/><br/>                      <p>                        <br>                  </p>                      <b>                        <font color="#292998">Integrating multidimensional omics data over biological networks with NetGestalt</font>                      </b>                      <br>                        <p>                          Jing Wang<sup>1</sup><br>                            <sup>1</sup>Vanderbilt University, Nashville, TN 37203, USA                          </p>                        <br>                          Biological networks have become a valuable model for the visualization and integration of different types of omics data. However, traditional node-link diagram-based network visualization becomes inadequate as network size and data complexity increase. This tutorial will introduce NetGestalt, a novel web-based data integration framework for addressing this challenge. NetGestalt transforms the traditional two-dimensional depiction of networks into a scalable one-dimension representation by exploiting the inherent hierarchical modular organization of biological networks. Thus, it allows simultaneous presentation of large-scale experimental and annotation data from various sources in the context of a biological network to facilitate data visualization, analysis, interpretation, and hypothesis generation. By the end of this tutorial, you should be able to understand the concept of the tool, prepare and upload your own data into NetGestalt, and perform different types of analysis in NetGestalt to generate hypothesis. <b>Recommended:</b> Prepare some omics data from your research (e.g. gene expression matrices, gene mutation profiles, t-statistics from differential expression analyses, or lists of interesting genes. Genes should be identified by gene symbols).<br/><br/>                          <td width="1.5%"></td>                            </tr>  </table>    -->                  <p>&nbsp;</p><p>&nbsp;</p><div id="clear"></div><div id="bottom"></div></div><div id="footer"></div><script>    $("#footer").load("footer.html");</script></body><!-- Mirrored from bioinfo.mc.vanderbilt.edu/icibm2013/tutorial.html by HTTrack Website Copier/3.x [XR&CO'2013], Mon, 07 Apr 2014 15:01:52 GMT --></html>